conformance/02_push_test.go

```diff
@@ -330,21 +330,28 @@ var test02Push = func() {
 		})
 
 		g.Context("Teardown", func() {
+			deleteManifest := func(req *reggie.Request) {
```

Comment:
Why not move this into setup.go or a utils.go/common.go
Other tests could use this too.